src/sidebar/components/TagEditor.js

```diff
@@ -75,11 +75,11 @@ function TagEditor({
    */
   const removeDuplicates = (suggestions, duplicates) => {
     const suggestionsSet = [];
-    suggestions.forEach(suggestion => {
+    for (let suggestion of suggestions) {
```

Comment:
Using a for loop instead of `forEach` enables the type of `suggestionsSet` to be inferred automatically.